ykshim_client/src/prod_api.rs

```diff
@@ -1,4 +1,5 @@
 //! The "Production" API client to ykshim.
+// XXX I'm not really sure what "production" means in this context.
```

Comment:
To be used in production as opposed to only used for testing I guess.

Reply:
Yes.

It's implied [here](https://github.com/softdevteam/yk/blob/d167d14fb9e026a45cf1ef77830a49008f3666e5/internal_ws/ykshim/src/lib.rs#L10), but we could make it more explicit.

Reply:
I think "production" is too vague a term to be useful. Can we find something that's clearer?

Reply:
We just need to find a term that means "not testing". Production is the best I can think of. Any suggestions?

Reply:
"Non-testing" might indeed be the best term?

Reply:
Hrm. Not sure about that. Do we really want files named 'non_testing.rs`? That's much less clear than `production` IMHO.

With the aid of a thesaurus, here's some more suggestions for "non testing api":
 - "essential API"
 - "real API"
 - "actual API"
 - "functional API"
 - "true API"

Reply:
They're all worse ;) I suspect the thing to do is to try and think of a different term than "production" entirely and that might then make thesaurus diving useful.

Reply:
"Working API" and then I'm spent.

Failing that, let's leave it as `production`. Software developers know that production is the opposite of testing. It makes sense.

Reply:
After an online discussion, I think we've agreed to remove the label "production" entirely on the basis that it's "normal" code. I *think* @vext01 also suggested renaming this to `lib.rs`. Testing modules will remain labelled as such to make clear that they're not "normal" code. @vext01 does that reflect your understanding?

Reply:
That's right. Just move the contents of `prod.rs` (in both workspaces) into the (already existing) `lib.rs`.

Reply:
https://github.com/softdevteam/yk/pull/260